pulseguardian/web.py

```diff
@@ -23,8 +23,7 @@
 
 from pulseguardian import config
 from pulseguardian.logs import setup_logging
-from pulseguardian.management import (PulseManagementAPI,
-                                      PulseManagementException)
+from pulseguardian import management as pulse_management
```

Comment:
cripes...  well, at least I was consistently foolish.  Hrm..  I remember that being called some kind of Hobgoblin...